sentry-spring/src/main/java/io/sentry/spring/SentryInitBeanPostProcessor.java

```diff
@@ -0,0 +1,21 @@
+package io.sentry.spring;
+
+import com.jakewharton.nopen.annotation.Open;
+import io.sentry.core.Sentry;
+import io.sentry.core.SentryOptions;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+
+/** Initializes Sentry after all beans are registered. */
```

Comment:
out of curiosity, if there's a bug on customer's beans and Sentry is the last to be inited, would that mean that we won't be able to capture the crash?

Reply:
This is why logging framework integration should be used as well. Logging framework is initialized first so it will catch any bug that happens during startup.

Reply:
ok sounds good, still curious how this would work, if a logging framework forwards the call to sentry, sentry won't be init./enabled yet, right, unless the logging framework has a sort of caching, not sure.
we can discuss this offline though, I'd like to understand a little bit more :) thanks.

Reply:
Logging frameworks calls init first, sends startup events to sentry, then Spring initializes Sentry and configuration provided there is used from now on. The drawback of this solution is that in order to get startup events sent to sentry DSN has to be provided twice - once to Logback configuration and then to Spring configuration.

Reply:
I see, but it's a good trade-off I'd say, we just document it properly, and it's fine IMO.
in the future we could look into it to provide a single DSN, might be possible with a little more work, just guessing here though.

Reply:
Yes, it will be possible if we follow up on https://github.com/getsentry/sentry-android/pull/536. To get it working SDK has to have a way to obtain configuration from external source - independent from framework-specific configuration. 
I think we will release what we have and then find out next steps from users feedback.

Reply:
that sounds great, thanks for explaining!